default_config.yml

```diff
@@ -0,0 +1,15 @@
+language: en
+
+pipeline: supervised_embeddings
```

Comment:
I think templates for pipelines no longer exist. They were deprecatated with Rasa 1.8 (https://rasa.com/docs/rasa/1.7.4/nlu/choosing-a-pipeline/#pre-configured-pipelines vs https://rasa.com/docs/rasa/1.8.3/nlu/choosing-a-pipeline/#pipeline-templates-deprecated)

Reply:
wait, that config is super old anyway 😆

Reply:
What is this even? 🤔 The default config I am actually using is in `rasa/configurations/default_config.yml`, no idea how this old thing snuck into the main folder 😆 will delete it